scripts/cc-cli/src/test/unit-tests/collect-coins.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  isExternalAddressValid,
+  isTxHashValid,
+} from "../../commands/collectCoins";
+
+describe("Test isTxHashValid", () => {
+  test("success", () => {
```

Comment:
"success" -> "should return true with a valid argument"  or something like that. Describes what the actual behavior we're looking for is.